.circleci/config.yml

```
@@ -639,14 +635,43 @@ jobs:
                 -DPIKA_WITH_STACKTRACES_STATIC_SYMBOLS=On \
                 -DPIKA_WITH_STACKTRACES_DEMANGLE_SYMBOLS=Off \
                 -DPIKA_WITH_TESTS_DEBUG_LOG=On \
-                -DPIKA_WITH_TESTS_DEBUG_LOG_DESTINATION=/build/debug-log-arm.txt \
+                -DPIKA_WITH_TESTS_DEBUG_LOG_DESTINATION=/build/debug-log.txt \
                 -DPIKA_WITH_SPINLOCK_DEADLOCK_DETECTION=On
-          working_directory: /home/circleci/project/build
       - run:
-          name: Build pika Arm64
+          name: Building pika Arm64
           command: |
+            cd build
             ninja -k 0 pika
+      - run:
+          name: Building tests Arm64
+          command: |
+            cd build
+            ninja -k 0 tests
+      - run:
+          name: Running tests Arm64
+          when: always
+          command: |
+            cd build
+            ulimit -c unlimited
+            ctest \
+              -j4 \
+              --timeout 400 \
+              -T test \
+              --no-compress-output \
+              --output-on-failure
+      - run:
+          <<: *convert_xml
+          working_directory: /home/circleci/project/build
+      - run:
+          <<: *move_core_dump
           working_directory: /home/circleci/project/build
+      - run:
+          <<: *move_debug_log
+          working_directory: /home/circleci/project/build
+      - store_test_results:
+          path: $HOME/project/build/tests
```

Can you use a relative path here? Environment variables do not seem to be expanded here.

```suggestion
          path: build/tests
```
?